az-core/src/main/java/azkaban/Constants.java

```diff
@@ -299,4 +313,9 @@
     public static final String START_TIME = "startTime";
     public static final String TRIGGER_INSTANCE_ID = "triggerInstanceId";
   }
+
+  public static class PageProperties {
+    // number of rows to be displayed on the executions page.
+    public static final String DISPLAY_EXECUTION_PAGE_SIZE = "displayExecutionPageSize";
```

Comment:
What's this for?

Reply:
@fwantastic The same comment as described for DISPLAY_EXECUTION_PAGE_SIZE? Could you please re-elaborate this constant?